stock_picking_invoicing_incoterm/__manifest__.py

```diff
@@ -1,21 +1,17 @@
-# Copyright 2014-2015 Agile Business Group sagl
+# Copyright 2020 Agile Business Group sagl
 # License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
 {
     "name": "Stock Picking Invoicing Incoterm",
-    "version": "8.0.1.0.0",
-    "category": "Warehouse Management",
+    "version": "14.0.1.0.0",
+    "category": "Hidden",
```

Comment:
Why `Hidden`? That is usually the category of auto-installing  modules.

The v14 equivalent of `Warehouse Management` might be `Inventory/Inventory`